src/Controls/src/Core/Button.cs

```diff
@@ -58,10 +58,7 @@ public Thickness Padding
 			set { SetValue(PaddingElement.PaddingProperty, value); }
 		}
 
-		Thickness IPaddingElement.PaddingDefaultValueCreator()
-		{
-			return default(Thickness);
-		}
+		Thickness IPaddingElement.PaddingDefaultValueCreator() => new Thickness(double.NaN);
```

Comment:
The most notable/breaking change. Default is no longer 0, but NaN to represent "OS values"